app/frontend/src/features/communities/CommunityPage/InfoPageSection.tsx

```diff
@@ -1,12 +1,12 @@
-import { Link as MuiLink, Typography } from "@material-ui/core";
+import { Link as MuiLink } from "@material-ui/core";
 import { InfoIcon } from "components/Icons";
+import Markdown from "components/Markdown";
 import { Community } from "proto/communities_pb";
 import { Link } from "react-router-dom";
 import { routeToCommunity } from "routes";
 import makeStyles from "utils/makeStyles";
 
 import { GENERAL_INFORMATION, SEE_MORE_INFORMATION } from "../constants";
-import getContentSummary from "../getContentSummary";
```

Comment:
Is this still used anywhere?

Reply:
Yeah looks like this might be unnecessary now as `PlacesCard` uses the `LineEllipsis` for doing the ...

I'll remove it!

Reply:
Oh wait, discussion still uses it